bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -105,7 +149,9 @@ Builder withQuery(Query query) {
      */
     @Deprecated
     public Builder withRequest(ReadRowsRequest request) {
-      return withRequest(StaticValueProvider.of(request));
+      ReadRowsRequest.Builder builder = request.toBuilder();
+      builder.setTableName(TableName.format(projectId.get(), instanceId.get(), tableId.get()));
```

Comment:
Please don't call get during pipeline construction time. Instead create another ValueProvider that takes these ValueProviders and resolves them when get() is called